scripts/extensions/pool_update.precheck

```diff
@@ -48,6 +48,14 @@ ERRORCODE = 'errorcode'
 ERROR = 'error'
 FOUND = 'found'
 REQUIRED = 'required'
+YUM_CMD = '/usr/bin/yum'
+DNF_CMD = '/usr/bin/dnf'
+PKG_MGR = DNF_CMD if os.path.exists(DNF_CMD) else YUM_CMD
```

Comment:
This is not using OO as it could: have a class that does what we need and zwo small sub-classes: one for yum and one for dnf. This would void the branching we are seeing here. It's probably ok because I don't expect more variation here.

Reply:
Yes, it can be. 
As you already point out, this is only tiny diff here between yum and dnf.
Even with class, we need branching which class to use anyway, so I prefer to just this none-OO styple, does this make sense? (If more and more variation comes, then we can use OO later.)